src/assets/json/mainnet.json

```diff
@@ -111,69 +111,79 @@
       "protocol": "http",
```

Comment:
These changes should go in `docs/assets/mainnet.json` .  I'm going to soon delete `src/assets/json/mainnet.json` to avoid confusion.